test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Sql/SchemaTests.cs

```diff
@@ -50,7 +50,7 @@ public async Task GivenAServerThatHasSchemas_WhenRequestingAvailable_JsonShouldB
             Assert.NotEmpty(jArrayResponse);
 
             JToken firstResult = jArrayResponse.First;
-            string scriptUrl = $"{_client.BaseAddress}_schema/versions/{firstResult["id"]}/script";
+            string scriptUrl = $"/_schema/versions/{firstResult["id"]}/script";
```

Comment:
should these test go to common code? and only have the FHIR specific ones here (if any?)

Reply:
Yes, good idea.

Reply:
Moved to common code in separate PR